armor-write/src/main/java/com/rapid7/armor/store/S3WriteStore.java

```diff
@@ -323,44 +324,30 @@ public void copyShard(ShardId shardIdDst, ShardId shardIdSrc) {
       return;
     }
 
+    String currentShardKey = null;
     try {
+      List<S3ObjectSummary> shardObjects = getShardObjects(shardIdSrc);
+      if (shardObjects.isEmpty()) {
+        throw new RuntimeException("Could not retrieve current contents of shard: " + shardIdSrc.shardIdPath());
+      }
+      
+      currentShardKey = shardSrcPath.resolve(shardSrcPath.relativize(Paths.get(shardObjects.get(0).getKey())).getName(0)).toString();
```

Comment:
Can you comment what like what it should look like this value. It is hard to visualize this line without stepping through.